delete_script.sql

```diff
@@ -0,0 +1,7 @@
+drop table appdata.vote;
+drop table appdata.poll;
+drop table appdata.user;
+DROP SEQUENCE poll_image_id_seq;
```

Comment:
I don't think we really need to add any sequence column here, I would prefer that our python script takes care of generating the  unique sequence that serves as the id for each row. We could do it like this:
```
import uuid
print(uuid.uuid4())
```

Reply:
We shall retain sequence just for image ids.

Reply:
lgtm